### PR TITLE
Add support for redhat 6 and rsyncssh

### PR DESCRIPTION
**Pull request by voxel01**
add support of sync type rsyncssh

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+    }
+    'Debian': {
+      case $::operatingsystem {
+        'Ubuntu': {
+          $config_file = "${config_dir}/lsyncd.conf.lua"
+				}
+		    default: {
+		      fail("Unsupported platform: ${module_name} currently doesn't support ${::osfamily} or ${::operatingsystem}")
+		    }
+		  }
```
Comment (negz) [inline manifests/params.pp:19]: This indenting looks a bit wonky.

**Comment by negz** [comment]
Nice! Sorry about the slow response - I only just noticed the PR.

I like the `lsyncd::sync::type` pattern, but it would be nice to keep `lsyncd::rsync` around as well for backwards compatibility for the time being.

**Comment by negz** [comment]
Ping. Happy to merge this if the comments are addressed.

**Comment by voxel01** [comment]
Is now adressed

**Pull request merged**